YAFC/Workspace/ProductionTable/ProductionTableView.cs

```diff
@@ -880,6 +880,10 @@ private enum ProductDropdownType {
             }
         }
 
+        private static bool CheckIfOverproducing(ProductionLink link) {
+            return link.algorithm == LinkAlgorithm.AllowOverProduction && (link.flags & ProductionLink.Flags.LinkNotMatched) != 0;
```

Comment:
There is a `HasFlags()` function to make checking for certain flags (slightly) nicer. See https://github.com/have-fun-was-taken/yafc-ce/pull/94/commits/27308c5136bfdb33b787696c0479933a2f296f8f#diff-75c10f5bf285559ced0cf39cf2285ef4f709280fb8fc34c50e1bfa57952a255aR827 for an example